src/main/resources/assets/jeresources/lang/ru_ru.json

```
@@ -1,129 +1,164 @@
+	"jer.dungeon.village_butcher": "Дом мясника",
+	"jer.dungeon.village_fletcher": "Мастерская лучника",
+	"jer.dungeon.village_fisher": "Хижина рыбака",
+	"jer.dungeon.village_tannery": "Кожевня",
+	"jer.dungeon.village_temple": "Церковь",
+	"jer.dungeon.village_desert_house": "Деревенский дом в пустыне",
+	"jer.dungeon.village_plains_house": "Деревенский дом в равнинах",
+	"jer.dungeon.village_taiga_house": "Деревенский дом в тайге",
+	"jer.dungeon.village_snowy_house": "Деревенский снежный дом",
+	"jer.dungeon.village_savanna_house": "Деревенский дом в саванне",
+	"jer.dungeon.underwater_ruin_small": "Малые подводные руины",
+	"jer.dungeon.underwater_ruin_big": "Большине подводные руины",
+	"jer.dungeon.buried_treasure": "Закопанное сокровище",
+	"jer.dungeon.shipwreck_map": "Затонувший корабль",
+	"jer.dungeon.shipwreck_supply": "Снабжение затонувшего корабля",
+	"jer.dungeon.shipwreck_treasure": "Сокровише затонувшего корабля",
+	"jer.dungeon.pillager_outpost": "Аванпост разбойников",
+	"jer.dungeon.bastion_treasure": "Сокровише бастиона",
+	"jer.dungeon.bastion_other": "Прочее бастиона",
+	"jer.dungeon.bastion_bridge": "Мост бастиона",
+	"jer.dungeon.bastion_hoglin_stable": "Загон хоглинов в бастионе",
+	"jer.dungeon.ruined_portal": "Разрушенный портал Нижнего мира",
 
-"_comment": "#Enchantments View",
-"jer.enchantments.title": "Зачарования",
+    "_comment": "#Enchantments View",
+    "jer.enchantments.title": "Зачарования",
 
-"_comment": "#Plant View",
-"jer.plant.title": "Плоды растений",
+    "_comment": "#Plant View",
+    "jer.plant.title": "Плоды растений",
 
-"_comment": "#Ore View",
-"jer.worldgen.title": "Генерация руд",
-"jer.worldgen.silkTouch": "Необходимо шёлковое касание",
-"jer.worldgen.biomes": "Появляется в биомах",
-"jer.worldgen.dimensions": "Допустимые измерения",
-"jer.worldgen.drops": "Выпадает",
-"jer.worldgen.average": "Сред. выпадение",
-"jer.worldgen.base": "Обычное",
+    "_comment": "#Ore View",
+    "jer.worldgen.title": "Генерация руд",
+    "jer.worldgen.silkTouch": "Необходимо шёлковое касание",
+    "jer.worldgen.biomes": "Появляется в биомах",
+    "jer.worldgen.dimensions": "Допустимые измерения",
+    "jer.worldgen.drops": "Выпадает",
+    "jer.worldgen.average": "Сред. выпадение",
+    "jer.worldgen.base": "Обычное",
+	"jer.worldgen.averageChunk": "Средн. блоков на чанк",
 
-"_comment": "#Mob View",
-"jer.mob.title": "Выпадение из мобов",
-"jer.mob.spawn": "Появляется в биоме:",
-"jer.mob.biome": "Появляется в биомах: наведите курсор сюда",
-"jer.mob.exp": "Даёт опыта",
+    "_comment": "#Mob View",
+    "jer.mob.title": "Выпадение из мобов",
+    "jer.mob.spawn": "Появляется в биоме:",
+    "jer.mob.biome": "Появляется в биомах: наведите курсор сюда",
+    "jer.mob.exp": "Даёт опыта",
 
-"_comment": "#Villager View",
-"jer.villager.title": "Торговля с крестьянином",
+    "_comment": "#Villager View",
+    "jer.villager.title": "Торговля с крестьянином",
```

```suggestion
    "jer.villager.title": "Торговля с жителем",
```
https://minecraft-ru.gamepedia.com/%D0%94%D0%B5%D1%80%D0%B5%D0%B2%D0%B5%D0%BD%D1%81%D0%BA%D0%B8%D0%B9_%D0%B6%D0%B8%D1%82%D0%B5%D0%BB%D1%8C